doc/liquid/tags.html

```diff
@@ -269,101 +122,27 @@ <h1>
-
 <p><strong>DEPRECATED</strong>: This tag is deprecated, use FlashDrop instead.</p>
-
 <p><strong>Example:</strong> Using flash tag in liquid</p>
-
 <pre lang="liquid"><code>&lt;html&gt;
   &lt;body&gt;
    {% flash %}
   &lt;/body&gt;
 &lt;/html&gt;
 </code></pre>
-
 <h1>
-<a name="tag-footer" class="anchor" href="#tag-footer"><span class="mini-icon mini-icon-link"></span></a>Tag 'footer'</h1>
-
+<a id="tag-footer" class="anchor" href="#tag-footer" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tag 'footer'</h1>
 <p>Renders a footer HTML snippet.</p>
-
 <p><strong>DEPRECATED</strong>: This tag is deprecated, use a CMS partial instead</p>
-
 <h1>
-<a name="tag-form" class="anchor" href="#tag-form"><span class="mini-icon mini-icon-link"></span></a>Tag 'form'</h1>
-
+<a id="tag-form" class="anchor" href="#tag-form" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tag 'form'</h1>
 <p>Renders a form tag with an action and class attribute specified, depending on the name
 of the form. The supported forms are:</p>
-
-<p></p><table>
-   <tr>
-     <th>Form</th>
-     <th>Allowed Field Names</th>
-     <th>Spam Protection</th>
-     <th>Notes</th>
-   </tr>
-   <tr>
-     <th>application.create</th>
-     <td>
-       <ul>
-         <li>application[name]</li>
-         <li>application[description]</li>
-         <li>application[&lt;any-extra-field&gt;]</li>
-       </ul>
-     </td>
-     <td>No</td>
-     <td></td>
-   </tr>
-   <tr>
-     <th>application.update</th>
-     <td>
-       <ul>
-         <li>application[name]</li>
-         <li>application[description]</li>
-         <li>application[&lt;any-extra-field&gt;]</li>
-       </ul>
-     </td>
-     <td>No</td>
-     <td></td>
-   </tr>
-   <tr>
-     <th>signup</th>
-     <td>
-       <ul>
-         <li>account[org_name]</li>
-         <li>account[org_legaladdress]</li>
-         <li>account[org_legaladdress_cont]</li>
-         <li>account[city]</li>
-         <li>account[state]</li>
-         <li>account[zip]</li>
-         <li>account[telephone_number]</li>
-         <li>account[country_id]</li>
-         <li>account[&lt;any-extra-field&gt;]</li>
-         <li>account[user][username]</li>
-         <li>account[user][email]</li>
-         <li>account[user][first_name]</li>
-         <li>account[user][last_name]</li>
-         <li>account[user][password]</li>
-         <li>account[user][password_confirmation]</li>
-         <li>account[user][title]</li>
-         <li>account[user][&lt;any-extra-field&gt;]</li>
-       </ul>
-     </td>
-     <td>Yes</td>
-     <td>Sign Up directly to plans of your choice by adding one
-         or more hidden fields with a name <code>plan_ids[]</code>.
-         If a parameter of such name is found in the current URL,
-         the input field is added automagically.
-     </td>
-   </tr>
- </table>
-
+<!-- raw HTML omitted -->
```

Comment:
I think this should not be omitted. Have you checked how it looks in the UI?

Reply:
wow, u r right! all of the sudden it is escaping this:
https://github.com/3scale/porta/blob/df1b415ad931e42ef5d9f51b88223076a74b6319/lib/developer_portal/lib/liquid/tags/form.rb#L14-L80

I didn't check the UI because I was expecting only 'string changes' introduced by ourselves in the descriptions... MY BAD! 🤦‍♀️ Gotta fix it! Good catch! 💪🏼 👏🏼

Reply:
Explained the mystery in [THREESCALE-6506](https://issues.redhat.com/browse/THREESCALE-6506) because the problem is only in filters.html and tags.html which I do not need to change. I do not have time for more. drops.md and drops.html are fine, which is what I need.